tools/jdbc/src/test/java/org/duckdb/test/TestDuckDBJDBC.java

```diff
@@ -410,7 +411,63 @@ public static void duckdb_timestamp_test() throws Exception {
 
 		conn.close();
 	}
+
+	public static void test_duckdb_localdatetime() throws Exception {
+		Connection conn = DriverManager.getConnection("jdbc:duckdb:");
+		Statement stmt = conn.createStatement();
+		stmt.execute("CREATE TABLE x (ts TIMESTAMP)");
+
+		LocalDateTime ldt = LocalDateTime.parse("2021-01-18T21:20:07");
```

Comment:
Will the instant specified by the string vary by testing time zone? Does it need an offset?

Reply:
The LocalDateTime internally consists of a LocalDate and a LocalTime. And these store the date/time just as ints. So there is no dependency on a testing time zone or offset that I could think of. 
I could make this more explicit by using a different constructor like: `LocalDateTime.of(2021,1,18,21,20,7)`

References:
https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/time/LocalDateTime.java
https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/time/LocalDate.java
https://github.com/openjdk/jdk/blob/master/src/java.base/share/classes/java/time/LocalTime.java

Reply:
Sounds good.